src/foam/u2/theme/StandaloneTheme.js

```diff
@@ -0,0 +1,66 @@
+/**
+* @license
+* Copyright 2024 The FOAM Authors. All Rights Reserved.
+* http://www.apache.org/licenses/LICENSE-2.0
+*/
+
+foam.CLASS({
+  package: 'foam.u2.theme',
+  name: 'StandaloneTheme',
+
+  documentation: `
+    A barebones theme object that can be used to theme non-nanos apps
+    Currently provides: 
+      - id for cssTokenOverrides
+      - name
+      - glyphs
+      - logos
+  `,
+
+
+  requires: [
+    'foam.u2.theme.ThemeGlyphs'
+  ],
+
+  properties: [
+    {
+      class: 'String',
+      name: 'id',
+      includeInDigest: true,
```

Comment:
Is this needed?

Reply:
But there will only be one of these objects in the system and it won't be stored in a DAO. Correct Sarthak?

Reply:
Yes, will update